ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -31,7 +31,7 @@ using std::ref;
 const double D2R = M_PI / 180.0 ;
 const double R2D = 180.0 / M_PI ;
 
-const double DegreeTolerance = 0.2;    // made up, degees
+const double DegreeTolerance = 0.4;    // made up, degees
```

Comment:
@mikedalal is the increase justified?

Reply:
I found in testing that .2 would sometimes trigger a timeout. Previously DegreeTolerance never mattered since the current degree and the goal degree were always set to the same value immediately and always returned as complete.

Reply:
.4 degree is very small, so I'm fine with this.  But let's document this for users.  At the moment the lander command/telemetry interface is documented entirely in plan-interface.h.